### PR TITLE
Fix Partial Serializer + Unassigened Transient issue

### PR DESCRIPTION
**Pull request by cdietrich**
Fix Partial Serializer + Unassigened Transient issue 
#1231

### CODE REVIEW

File: org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/PartialSerializer.java [hunk 1]
```diff
@@ -336,6 +336,10 @@ public void setErrorAcceptor(ISerializationDiagnostic.Acceptor errorAcceptor) {
 		IFeatureInfo featureInfo = constraint.getFeatures()[owner.eClass().getFeatureID(feature)];
 		List<IConstraintElement> assignments = featureInfo.getAssignments();
 		if (assignments.size() != 1) {
+			ValueTransient valueTransient = transientValues.isValueTransient(owner, feature);
```
Comment (szarnekow) [inline org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/PartialSerializer.java]: What happens if there is more than one assignment? E.g. should the check for valueTransient be enhanced by `&& assignments.isEmpty()`?

Comment (cdietrich) [reply org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/PartialSerializer.java]: Good question. Have to think about that

Comment (cdietrich) [reply org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/PartialSerializer.java]: yes you are right. transient and zero assignments is fine. other cases really might need the sequencer to kick in

Comment (cdietrich) [reply org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/PartialSerializer.java]: updated the pr

**Comment by cdietrich** [comment]
@meysholdt if you find the time to review somewhen it would be nice

**Pull request merged**